models/index.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+
+module.exports = function(sequelize) {
+    fs.readdirSync('./models')
+        .filter(file => file.slice(-3) === '.js')
+        .filter(file => file !== 'baseModel.js')
+        .filter(file => file !== 'index.js')
```

Comment:
Maybe instead use a special suffix (like how *.spec.js works for tests) or another subdirectory?

Reply:
the special suffix feels a little weird...i figured that since this logic was hidden away in an an index for that directory, it was ok that it had some 'specific' knowledge about what should and shouldn't be included.  do you think it's worth changing?